src/olympia/abuse/models.py

```diff
@@ -33,14 +68,51 @@ class AbuseReport(ModelBase):
         on_delete=models.SET_NULL)
     message = models.TextField()
 
+    # Extra optional fields for more information, giving some context that is
+    # meant to be extracted automatically by the client (i.e. Firefox) and
+    # submitted via the API.
+    client_id = models.CharField(
+        default=None, max_length=64, blank=True, null=True)
+    addon_name = models.CharField(
+        default=None, max_length=255, blank=True, null=True)
```

Comment:
Are these so long max lengths because we usually don't care? Addon name is currently limited to 50 characters in the devhub form for example.

Reply:
The add-on doesn't have to be listed, and it could have been installed from before the limits were put in place in devhub anyway. I didn't have any direction on this so I just picked the standard largish value.

Reply:
Sounds fair :+1: